pymeasure/display/windows.py

```diff
@@ -497,3 +497,396 @@ def finished(self, experiment):
         if not self.manager.experiments.has_next():
             self.abort_button.setEnabled(False)
             self.browser_widget.clear_button.setEnabled(True)
+
+# JM: ############################################################################
+
+# QUESTION: Can we just inherit from ManagedWindow?
+class ManagedImageWindow(QtGui.QMainWindow):
+    """
+    Abstract base class.
+
+    The ManagedWindow provides an interface for inputting experiment
```

Comment:
This should read `ManagedImageWindow`. Please do a search/replace in this section.